### PR TITLE
feat: add rangeUntil operator

### PR DESCRIPTION
**Pull request by sarahhaggarty**
This PR updates our Ranges page with the `..<` operator. It also adds this operator the list of keywords and operators.

### CODE REVIEW

**Review by zarechenskiy — CHANGES_REQUESTED** [review]

File: docs/topics/ranges.md [hunk 1]
```diff
@@ -1,119 +1,109 @@
...
 
 ```kotlin
-
-fun main() {
+fun main() { 
 //sampleStart
-    for (i in 1..4) print(i)
+    val i = 4 
+
+    if (i in 1.rangeTo(4)) { // equivalent to i >= 1 && i <= 4
```
[the remark this answers is not in the document]
Comment (sarahhaggarty) [reply docs/topics/ranges.md:29]: Guilty as charged! Fixed.

File: docs/topics/ranges.md [hunk 2]
```diff
@@ -1,119 +1,109 @@
...
+`kotlin.ranges` package.
+
+To create a closed-ended range, use the `.rangeTo()` extension function:
 
 ```kotlin
 fun main() {
     val i = 1 
 //sampleStart
-    if (i in 1..4) { // equivalent of i >= 1 && i <= 4
+    if (i in 1.rangeTo(4)) { // equivalent to i >= 1 && i <= 4
```
[the remark this answers is not in the document]
Comment (zarechenskiy) [reply docs/topics/ranges.md]: I think it's pretty clear now, thank you :)

File: docs/topics/keyword-reference.md [hunk 3]
```diff
@@ -147,7 +147,7 @@ Kotlin supports the following operators and special symbols:
  * `?.` performs a [safe call](null-safety.md#safe-calls) (calls a method or accesses a property if the receiver is non-null).
  * `?:` takes the right-hand value if the left-hand value is null (the [elvis operator](null-safety.md#elvis-operator)).
  * `::` creates a [member reference](reflection.md#function-references) or a [class reference](reflection.md#class-references).
- * `..` creates a [range](ranges.md).
+ * `..`, `..<` creates [ranges](ranges.md).
```
Comment (danil-pavlov) [inline docs/topics/keyword-reference.md:150]:
```suggestion
 * `..`, `..<` create [ranges](ranges.md).
```

File: docs/topics/ranges.md [hunk 4]
```diff
@@ -1,51 +1,54 @@
 [//]: # (title: Ranges and progressions)
 
-Kotlin lets you easily create ranges of values using the [`rangeTo()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.ranges/range-to.html)
-function from the `kotlin.ranges` package and its operator form `..`. Usually, `rangeTo()` is complemented by `in` or
-`!in` functions.
+Kotlin lets you easily create ranges of values using the [`.rangeTo()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.ranges/range-to.html)
+and [`.rangeUntil()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.ranges/range-until.html) functions from the 
+`kotlin.ranges` package. The easiest way to call these functions is by using their operator forms `..` and `..<` respectively.
+
+To create a closed-ended range, call the `.rangeTo()` function with the `..` operator.
```
Comment (danil-pavlov) [inline docs/topics/ranges.md:7]:
```suggestion
To create a closed-ended range, call the `rangeTo()` function with the `..` operator.
```

Comment (danil-pavlov) [inline docs/topics/ranges.md:7]:
```suggestion
To create a closed-ended range, call the `rangeTo()` function with the `..` operator.
```

File: docs/topics/ranges.md [hunk 5]
```diff
@@ -1,51 +1,54 @@
 [//]: # (title: Ranges and progressions)
 
-Kotlin lets you easily create ranges of values using the [`rangeTo()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.ranges/range-to.html)
-function from the `kotlin.ranges` package and its operator form `..`. Usually, `rangeTo()` is complemented by `in` or
-`!in` functions.
+Kotlin lets you easily create ranges of values using the [`.rangeTo()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.ranges/range-to.html)
```
Comment (danil-pavlov) [inline docs/topics/ranges.md:3]:
```suggestion
Kotlin lets you easily create ranges of values using the [`rangeTo()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.ranges/range-to.html)
```

File: docs/topics/ranges.md [hunk 6]
```diff
@@ -1,51 +1,54 @@
...
+and [`.rangeUntil()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.ranges/range-until.html) functions from the
```
Comment (danil-pavlov) [inline docs/topics/ranges.md:4]:
```suggestion
and [`rangeUntil()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.ranges/range-until.html) functions from the 
```

**Review by danil-pavlov — COMMENTED** [review]
I think it's worth adding the new operator to the [Idioms](https://kotlinlang.org/docs/idioms.html#iterate-over-a-range) page

**Review by danil-pavlov — APPROVED** [review]

**Pull request merged**